.github/workflows/python-package.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    paths-ignore:
+      - 'docs/**'
+  pull_request:
+    branches: *
```

Comment:
Minor - shouldn't this be `'*'` for satisfying yaml syntax?

Reply:
that might explain why the workflow failed :)